Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -160,53 +158,50 @@ Constructor post-processing:
 Hook configuration
 ==================
 
-There is no complete index of hooks in the Core. But they are easy to
-search for and find. And typically it comes quite naturally since you
-will find the hooks in the code you want to extend - if they exist.
-
-This index will list the main variable spaces for configuration of
-hooks. By the names of these you can easily scan the source code to
-find which hooks are available or might be interesting for you.
-
-The index below also includes some variable spaces which not only
-carry hook configuration but might be used for other purposes as well.
+Most hooks in the TYPO3 Core have been converted into events which are
+completely listed in the :ref:`event list <eventlist>`.
 
+There is no complete index of the remaining hooks in the Core. The following
+naming scheme is usually used by third-party extensions and the remaining
+Core hooks:
```

Comment:
Why a colon? Hooks from core are mostly available under SC_OPTIONS not EXTCONF?

Reply:
"third-party extensions and the remaining Core hooks:"